.changeset/brown-buttons-burn.md

```diff
@@ -0,0 +1,6 @@
+---
+"@janus-idp/backstage-plugin-orchestrator-form-react": patch
+"@janus-idp/backstage-plugin-orchestrator-form-api": patch
+---
+
+fix several issues
```

Comment:
The changelog will look beter with additional details :-)

Reply:
fixed